packages/api/app/Controllers/Http/KnowledgeAreaController.js

```diff
@@ -0,0 +1,19 @@
+const KnowledgeArea = use('App/Models/KnowledgeArea');
+
+class KnowledgeAreaController {
+	async show({ params }) {
+		const { knowledge_area_id } = params;
+		const knowledgeArea = await KnowledgeArea.getKnowledgeArea(knowledge_area_id);
+		await knowledgeArea.loadMany(['greatArea', 'area', 'subArea', 'speciality']);
+		return knowledgeArea;
+	}
```

Comment:
```suggestion
	async show({ request }) {
		request.params.id = request.params.knowledge_area_id;
		return KnowledgeArea.query().withParams(request);
	}
```
Ou poderia mudar `knowledge_area_id` para `id`

Reply:
@NaylsonFerreira eu até pensei em colocar id, mas para seguir o padrão eu deixei um campo personalizado, além disso ele é biginteger e não increment